modules/javafx.graphics/src/main/java/javafx/application/Preloader.java

```diff
@@ -110,8 +107,7 @@ public abstract class Preloader extends Application {
     private static final String lineSeparator;
```

Comment:
```suggestion
    private static final String lineSeparator = System.getProperty("line.separator", "\n");
```

Reply:
I would suggest to remove the variable `lineSeparator` and simply use `System.lineSeparator()`.

Reply:
I agree. That has been available since 1.7. But possibly this code is old enough to pre-date that ..

Reply:
This code possibly dates from JDK 6. If we want to change it (which we probably should), let's do this as a follow-up, since all of the doPrivileged PRs are intended to be behavior neutral.

Reply:
Done (there actually even was a comment right above this about support for `lineSeparator()` in core)